tests/plugins/indentlist/manual/taberrorininlineeditor.md

```diff
@@ -0,0 +1,12 @@
+@bender-tags: 4.7.1, tc, 424
```

Comment:
Update tags to the new bug/feature convention.

Additionally this test does not make much sense in browsers other than Chrome (as this thing was always working there) – let's run it only in Chrome.